src/components/Process/Chained.tsx

```diff
@@ -274,7 +317,11 @@ type FlowCondition = {
   goto: string
 }
 
+// FlowNode can have or conditions or renderWith, but not both
 type FlowNode = {
   default: string
-  conditions?: FlowCondition[]
+} & ({ conditions?: FlowCondition[]; renderWith?: never } | { conditions?: never; renderWith: RenderWith[] })
+
+export type RenderWith = {
+  id: string
 }
```

Comment:
Shouldn't be required to do this, since it's already declared in ui-components (use that instead).

Reply:
02c95a3